rusk-wallet/src/lib/prompt.rs

```diff
@@ -441,7 +441,7 @@ fn request_rcvr_addr() -> String {
 
 /// Utility function to check if an address is valid
 fn is_valid_addr(addr: &str) -> bool {
-    bs58::decode(addr).into_vec().is_ok() && !addr.is_empty()
+    !addr.is_empty() && bs58::decode(addr).into_vec().is_ok()
```

Comment:
This change is unrelated to this PR but was suggested [here](https://github.com/dusk-network/rusk/pull/607#discussion_r816779808)